### ISSUE TITLE
The record button is not accessible with screen reader.

### ISSUE THREAD

**Issue by chigkim** [Joooohan/audio-recorder-streamlit#20]
Hi,

It looks like the record button is not accessible with screen readers.

The problems are:
1. It seems clicking the svg element triggers the recording.
2. The element has aria-hidden="true" which means telling assistive technology such as screen reader to ignore the element.
3. The element also has focusable="false" which prevents users from focusing on the element with keyboard like tab/shift+tab and press enter/space to activate.

If I do this, it works.
1. remove aria-hidden attribute.
2. set focusable="true".
3. Add role="button".
4. Add aria-label="Record".

Could you investigate?
Thanks so much!

**Comment by Joooohan** [comment]
Hello, I have added a proper button element which is focusable and has an aria-label. I will release it in 0.0.10.

**Comment by Joooohan** [comment]
I have released 0.0.10.
